.github/workflows/ci.yaml

```diff
@@ -18,15 +37,15 @@ jobs:
         channels: conda-forge,bioconda
         channel-priority: true
         activate-environment: test
-    - run: mamba install mafft raxml fasttree iqtree vcftools pip numpy
-    - run: pip install biopython==1.67
+    - run: mamba install mafft raxml fasttree iqtree vcftools
+    - run: mamba install biopython=${{ matrix.biopython-version }}
     - run: pip install -e .[dev]
     - run: conda info
     - run: conda list
     - run: pytest -c pytest.python3.ini  --cov-report=xml --cov=augur
     - run: cram --shell=/bin/bash tests/functional/*.t tests/builds/*.t
     - run: bash tests/builds/runner.sh
-    - if: github.repository == 'nextstrain/augur' && matrix.python-version == 3.10
+    - if: github.repository == 'nextstrain/augur' && matrix.versions.python == 3.10
```

Comment:
Oh, just noticed this needs switching back to `matrix.python-version`.

Reply:
Oof, good catch. Fixed https://github.com/nextstrain/augur/pull/843/commits/d068c9715484e2953016b4a221e8289417efe74b

Reply:
Oh, and just also realized that it needs to condition on `matrix.biopython-version` since there are two jobs that run where `python-version == 3.10`.

Also, this version comparison probably need to be `== '3.10'` instead of comparing a string to float?

Reply:
Right... thanks 😅 https://github.com/nextstrain/augur/pull/843/commits/456b65a83965c837406cf9d206e779d8f00eb728

Reply:
Oddly, codecov bot posted the comment for https://github.com/nextstrain/augur/commit/d068c9715484e2953016b4a221e8289417efe74b though the codecov step was skipped on all jobs in that workflow run.

Reply:
I think this is resolved, but the condition is getting hairy. Another reason for #845